runtime/syntax/asciidoc.vim

```diff
@@ -25,6 +25,25 @@ if exists("b:current_syntax")
   finish
 endif
 
+" Conceal Systems
+let s:conceal = ''
+let s:concealends = ''
+let s:concealcode = ''
+if has('conceal') && get(g:, 'vim_asciidoc_conceal', 1)
+  let s:conceal = ' conceal'
+  let s:concealends = ' concealends'
+endif
+if has('conceal') && get(g:, 'vim_asciidoc_conceal_code_blocks', 1)
```

Comment:
missing documented variable

Reply:
if let g:vim_asciidoc_conceal_conceal_code_block=0 vim will not activated conceal in specific code block like `[] _ *`.